dev/src/bulk-writer.ts

```diff
@@ -107,16 +121,24 @@ class BulkWriterOperation {
     return this.deferred.promise;
   }
 
+  get lastError(): Status | undefined {
```

Comment:
I am surprised this need to be exported.

Reply:
Me too, removed.